README.md

```diff
@@ -30,6 +30,27 @@ Running the tests
 
 `docker-compose run cli vendor/bin/phpunit`
 
+Running Behat
+-------------
+
+Behat needs the `ci` image to run, so it needs to build an additional image and use the ci configuration:
```

Comment:
Should be possible to use the `cli` image, so can avoid `docker-compose.ci.yml` (means the `selenium` image has to be moved/duplicated etc).

Reply:
Bit more complicated than first thought as there needs to be a `ci` FPM container as well. Not sure it's worth changing.